src/pages/[id].page.test.jsx

```diff
@@ -0,0 +1,7 @@
+import '@testing-library/jest-dom';
+// import { render, screen } from '@testing-library/react';
+// import { act } from 'react-dom/test-utils';
```

Comment:
More zombie code 🧟 
In fact, this whole file isn't really doing anything super useful, so I'd be fine with removing it entirely. The individual components that make up the `[id]` page should be tested well enough to give us confidence that the whole page will work (and it's easy enough to check with our limited number of pages).